styles.css

```diff
@@ -0,0 +1,995 @@
+  font-weight: 900;
+}
+
+.partner {
+  background-color: #424141;
+  color: #fff;
+  padding: 2rem;
+}
+
+.partner-title {
+  text-align: center;
+  font-size: 1.3rem;
+}
+
+.bar-footer {
+  background-color: #ec5242;
+  width: 2rem;
+  height: 0.03rem;
+  margin: auto auto 1.5rem;
+}
+
+.footer-partners {
+  list-style: none;
+  padding: 0;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  gap: 1.5rem;
+}
+
+.footer-item a {
+  text-decoration: none;
+  color: #d3d3d3;
+}
+
+.footer-item a:hover {
+  color: #fff;
+}
+
+.partner1 {
+  font-size: 1.5rem;
+}
+
+.partner2 {
+  font-size: 1.25rem;
+  font-weight: 300;
+  letter-spacing: 0.1rem;
+}
+
+.partner3 {
+  font-size: 2rem;
+  font-weight: 800;
+}
+
+.partner4 {
+  font-size: 1.1rem;
+  letter-spacing: 0.1rem;
+}
+
+.partner5 {
+  font-size: 1.7rem;
+}
+
+.copyright {
+  display: flex;
+  padding: 1.5rem;
+}
+
+.logo-footer {
+  color: #ec5242;
+  flex: 1;
+  margin: auto;
+  text-align: center;
+  font-size: 1rem;
+  letter-spacing: 0.2rem;
+}
+
+.rights-declaration {
+  font-size: 0.8rem;
+  color: #424141;
+  margin: 0;
+}
+
+.rights {
+  flex: 1;
+}
+
+.rights-declaration-desktop {
+  display: none;
+}
+
+.bar-about-main {
+  display: none;
+}
+
+.speakers {
+  padding: 2rem;
+}
+
+.featured {
+  text-align: center;
+  font-weight: 900 !important;
+}
+
+.footer-home {
+  display: none;
+}
+
+.card {
+  border: none !important;
+  gap: 2rem !important;
+  margin: 3rem 0;
+}
+
+.featuredSpeakers {
+  display: flex;
+}
+
+.cardImg {
+  width: 90px;
+  height: 90px;
+}
+
+.speakersInfo {
+  margin-left: 2rem;
+}
+
+.speakerName {
+  font-size: 1.2rem;
+  font-weight: 700;
+}
+
+.speakerPosition {
+  color: #ec5242;
+  font-style: italic;
+  font-size: 0.9rem;
+}
+
+.speakerBar {
+  background-color: #d3d3d3;
+  width: 3rem;
+  height: 0.1rem;
+  margin: 1rem 0;
+}
+
+.speakerExperience {
+  color: #474747;
+  font-size: 0.9rem;
+}
+
+.more-button {
+  width: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: #8b8787 solid 1px;
+  border-radius: 0.25rem;
+  background-color: #fff;
+}
+
+.more-button-p {
+  margin: 0;
+  padding: 1rem 2rem;
+}
+
+.card-hidden {
+  display: none !important;
+}
+
+@media only screen and (max-width: 767px) {
+  .program {
+    display: none;
+  }
+}
+
+/* desktop media querie */
+@media only screen and (min-width: 768px) {
+  .intro-main {
+    background-image: url(Images/bg.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    border: #fff 1px solid;
+    position: relative;
+  }
+
+  .intro-about {
+    background-image: url(Images/about_bg.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    border: #fff 1px solid;
+    position: relative;
+  }
+
+  .wttlogo {
+    width: 280px;
+  }
+
+  .intro {
+    padding: 0 0 5rem;
+    width: 70%;
+    background-repeat: no-repeat;
+    background-size: cover;
+  }
```

Comment:
- [ ] Please make sure to style the about page according to the design guidelines as suggested by the previous reviewer [here](https://github.com/gdvl777/CapstoneSmash/pull/1#discussion_r1202890551). The introduction elements of about page are still on left 🔨 This can be fixed by using `margin: 0 auto;` for desktop  #version😎🤓
- [ ] Also, I think it would be better to use the following for the background image of the desktop version 🔨💡 so that the page looks almost identical to the original design as suggested by the reviewer

```
  background-position: center;
  background-size: contain;
```